plugin/trino-exchange-filesystem/src/main/java/io/trino/plugin/exchange/filesystem/s3/ExchangeS3Config.java

```diff
@@ -270,4 +277,31 @@ public ExchangeS3Config setGcsJsonKey(String gcsJsonKey)
         this.gcsJsonKey = Optional.ofNullable(gcsJsonKey);
         return this;
     }
+
+    @NotNull
+    public S3SseType getSseType()
+    {
+        return sseType;
+    }
+
+    @Config("exchange.s3.sse.type")
+    @ConfigDescription("Type of S3 server-side encryption for FTE")
+    public ExchangeS3Config setSseType(S3SseType sseType)
+    {
+        this.sseType = sseType;
+        return this;
+    }
+
+    public String getSseKmsKeyId()
```

Comment:
Can we have a validation here - like with `AssertTrue` annotation - so that KmsKeyId is only set then the `sseType` is `KMS`